app/src/components/assets/AssetType.tsx

```diff
@@ -0,0 +1,152 @@
+import { IonCard, IonCardContent, IonCardHeader, IonCardSubtitle, IonCardTitle, IonCol, IonImg, IonItem, IonLabel, IonList, IonRow, IonSlide, IonSlides, useIonViewWillLeave } from "@ionic/react";
+import { useEffect, useRef, useState } from "react";
+import { useParams } from "react-router";
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import Api from "../../controllers/Api";
+import { fileExtensionToIcon, formatSize, s3url } from "../../globals/functions";
+import Page from "../../pages/Page";
+import { faArrowRight, faBan, faFlag } from "@fortawesome/free-solid-svg-icons";
+import axios from "axios";
+import { AssetTypeData } from "../../globals/interfaces";
+
+const AssetType = () => {
+    let { type } = useParams<{type: string}>();
+    const [assetType, setAssetType] = useState<AssetTypeData>({tags: [], thumnails: [], files: [], fields: []});
+    let cancelToken = axios.CancelToken.source();
+
+    //get data
+    useEffect(() => {
+        async function getAssets() {
+            setAssetType({tags: [], thumnails: [], files: [], fields: []});
+            const fetchedAssets = await Api("assets/list.php", {"assetTypes_id": type,"all": true}, cancelToken.token);
+            try {
+                setAssetType(fetchedAssets['assets'][0]); //as this is only listing one type, only return one object
+            } catch (error) {
+                console.log("[AdamRMS] " + error );
```

Comment:
possibly add sentry or is it too early?

Reply:
imho way too early, this was mainly a dev log as in other asset files

Reply:
Happy with that